pmlc/dialect/stdx/transforms/BUILD

```diff
@@ -24,13 +24,19 @@ plaidml_cc_library(
     srcs = [
         "boundscheck.cc",
         "i1_to_i32.cc",
+        "subgroup_broadcast.cc",
         "pass_detail.h",
     ],
     hdrs = [
         "passes.h",
     ],
     deps = [
         ":gen-pass",
+        "@llvm-project//mlir:SCFDialect",
+        "@llvm-project//mlir:GPUDialect",
+        "@llvm-project//mlir:GPUTransforms",
+        "@llvm-project//mlir:VectorOps",
```

Comment:
Would be good to keep other dialects out of this one. If we need to perform a conversion from one dialect to another, we should put it in the `conversion` directory.

Reply:
I can do that, will create vector_to_stdx directory and move the implementation there. I will do this at later step